_config.yml

```diff
@@ -100,6 +100,6 @@ playbook:
    - anchor: security
      img: graphic-hex
      text: Create responsive security systems
-baseurl:
+baseurl: 
```

Comment:
since the href uses baseurl, this won't update until baseurl is defined. 

I think we want `href="{{ '' | absolute_url }}"` -- @zachmargolis is that correct?

Reply:
or `href="{{ '/'| absolute_url }}"` ? might have to try em out

Reply:
Since this is all whitespace, YAML will just turn this into `nil` .... I'd undo this change

Also I tend to use `{ '/url' | prepend: site.baseurl }` --- I'll push a commit with these changes

Reply:
Thanks all!